pages.da/common/ncdu.md

```diff
@@ -0,0 +1,24 @@
+# ncdu
+
+> Diskforbrugsanalyse i ncurses interface.
```

Comment:
I checked this line with Google translate. And it wasn't displaying properly for me. Can this be used?

```suggestion
> Diskbrugsanalysator med en ncurses-grænseflade.
```